src/current/v24.1/cutover-replication.md

```diff
@@ -164,13 +164,13 @@ To move back to a different cluster, follow the physical cluster replication [se
 
 ### Example
 
-The following is a step-by-step example of cutting back to the original primary cluster. This example assumes that a cutover from a primary to standby cluster has occurred, and the promoted standby has been serving traffic.
+This section illustrates the steps to cut back to the original primary cluster from the promoted standby cluster that is currently serving traffic.
 
 - **Cluster A** = original primary cluster
 - **Cluster B** = original standby cluster
 
-1. **Cluster B** is now serving application traffic after the [cutover](#step-2-complete-the-cutover).
-1. To begin the cutback to the original primary cluster (**cluster A**), it is necessary that the virtual cluster (e.g., `vc_a`) is no longer serving traffic. Connect to the system virtual on **cluster A**:
+1. **Cluster B** is serving application traffic after the [cutover](#step-2-complete-the-cutover).
```

Comment:
Nit: It doesn't seem quite right that this is a numbered step. Maybe it should be a paragraph after the list.